utils/Theme.ts

```diff
@@ -0,0 +1,21 @@
+import { PaletteColorOptions } from '@mui/material';
+import { TypeBackground } from '@mui/material/styles/createPalette';
+
+export const primary: PaletteColorOptions  = {
+    light: "#ffffff",
+    main: "#ffffff",
+    dark: "#cccccc",
```

Comment:
When not hovering the button is like that :
![image](https://user-images.githubusercontent.com/71513074/145579516-4a37d6b4-dbec-491a-99a1-ea14d6e3fc64.png)
But when hovering the button is like that :
![image](https://user-images.githubusercontent.com/71513074/145579605-bf789fb7-df5d-4190-b3ae-3638d7fceff4.png)

I don't think it makes sense to be like that, so i propose to replace this to make it like that when hovering :
![image](https://user-images.githubusercontent.com/71513074/145579692-26671988-fa12-403b-a28e-4cd289ab729d.png)

```suggestion
    dark: "#3d19a3",
```

Reply:
I proposed a better change :
```tsx
<Button variant="contained" sx={{ color: "primary.main", bgcolor: "secondary.main", '&:hover': { color: "primary.main", bgcolor: 'secondary.dark' }}}>Login</Button>
```